hugr-model/src/v0/text/print.rs

```diff
@@ -460,25 +446,37 @@ impl<'p, 'a: 'p> PrintContext<'p, 'a> {
         }
     }
 
+    fn print_params(&mut self, params: &'a [Param<'a>]) -> PrintResult<()> {
+        for param in params {
```

Comment:
nit: can you just `params.iter().try_for_each(self.print_param)` (maybe `.copied()` or something) ?